maple-sdk/introduction.md

```diff
@@ -0,0 +1,33 @@
+# Introduction
+
+`maple-js` is a comprehensive and user-friendly JavaScript SDK for Maple Protocol. It is designed to enable seamless interaction with Maple Protocol's smart contracts on Ethereum. This SDK equips developers with the essential tools to integrate with the Maple Protocol ecosystem, simplifying the process of connecting to the blockchain, managing accounts, submitting transactions, and handling events generated by smart contracts.
+
+Utilizing the JavaScript SDK for Maple Protocol allows developers to efficiently create robust and secure dApps that leverage the full potential of Maple Protocol's smart contracts, fostering innovation and driving the adoption of decentralized finance solutions.
+
+Whether you are building backend services, frontend applications, or a combination of both, the Maple SDK provides the tools you need to create a seamless user experience.
+
+# Exports
+
+| Export                 | Purpose                                                                                 |
```

Comment:
What is the sort reasoning here? Seems like maybe we should do alphabetical OR logical. By logical I mean in a way that the read could potentially understand the building blocks of the system as they are introduced. So maybe globals, pool stuff, loan stuff, etc